src/node_jsvmapi.cc

```diff
@@ -514,6 +514,10 @@ namespace v8impl {
 #define CHECK_NEW_FROM_UTF8(isolate, result, str)                       \
     CHECK_NEW_FROM_UTF8_LEN((isolate), (result), (str), -1)
 
+#define GET_RETURN_STATUS()                                             \
+  v8impl::TryCatch::lastException().IsEmpty()                           \
+    ? napi_ok : napi_pending_exception
```

Comment:
Wrap the whole macro expansion in parentheses in case it is used in a larger expression.